cabal-install/bootstrap.sh

```diff
@@ -181,8 +181,18 @@ PARSEC_VER="3.1.9";    PARSEC_VER_REGEXP="[3]\.[01]\."
                        # >= 3.0 && < 3.2
 DEEPSEQ_VER="1.4.2.0"; DEEPSEQ_VER_REGEXP="1\.[1-9]\."
                        # >= 1.1 && < 2
-BYTESTRING_VER="0.10.8.1"; BYTESTRING_VER_REGEXP="0\.10\.[2-9]"
-                       # >=0.10.2
+
+case "$GHC_VER" in
```

Comment:
Hmm, won't we have to repeat the conditional or at least conditionalise the do_pkg and info_pkg calls as well?